qlib/data/data.py

```diff
@@ -617,7 +617,7 @@ def inst_calculator(inst, start_time, end_time, freq, column_names, spans=None,
         for _processor in inst_processors:
             if _processor:
                 _processor_obj = init_instance_by_config(_processor, accept_types=InstProcessor)
-                data = _processor_obj(data)
+                data = _processor_obj(data, instrument=inst)
```

Comment:
Thanks for your advice!  It looks more reasonable to add this parameter.

In addition, I think the interface should be changed with it
https://github.com/microsoft/qlib/blob/main/qlib/data/inst_processor.py#L8